classes/02class/exercises/c04-iam01/jayanath/cli_commands.txt

```diff
@@ -0,0 +1,72 @@
+## Command Execution Output
+```
+> aws iam create-user --user-name user_readonly_S3
+    {
+        "User": {
+            "Path": "/",
+            "UserName": "user_readonly_S3",
+            "UserId": "AIDA5GMYZ45KCWKCMXDUP",
+            "Arn": "arn:aws:iam::907095435092:user/user_readonly_S3",
+            "CreateDate": "2020-06-29T10:39:20+00:00"
+        }
+    }
+
+## Create a file with policy json with below content.
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:Get*",
+                    "s3:List*"
+                ],
+                "Resource": "*"
+            }
+        ]
+    }
+
+> aws iam create-policy --policy-name S3readOnly --policy-document file://S3readOnly.json
+    {
+        "Policy": {
+            "PolicyName": "S3readOnly",
+            "PolicyId": "ANPA5GMYZ45KNTHTQPBEY",
+            "Arn": "arn:aws:iam::907095435092:policy/S3readOnly",
+            "Path": "/",
+            "DefaultVersionId": "v1",
+            "AttachmentCount": 0,
+            "PermissionsBoundaryUsageCount": 0,
+            "IsAttachable": true,
+            "CreateDate": "2020-06-29T11:34:32+00:00",
+            "UpdateDate": "2020-06-29T11:34:32+00:00"
+        }
+    }
+
+> aws iam attach-user-policy \
+    --user-name user_readonly_S3 \
+    --policy-arn arn:aws:iam::907095435092:policy/S3readOnly
+
+## below access key and secret access key were deleted before committing the code :-)
```

Comment:
exercise looks good but do you mind deleting the secrets from it please?
I know there are not valid but github keeps throwing warnings because of that

thanks!

Reply:
Thanks @caiocezart. Removed the keys.